### PR TITLE
dsl: add sdformat install_bottle jobs, add ign-utils to edifice view

### PR DESCRIPTION
**Pull request by scpeters**
We noticed in https://github.com/osrf/homebrew-simulation/pull/1458 that there were not `install_bottle` jobs for sdformat and that the `ign-utils` install jobs weren't included in the edifice view. This remedies both of those things and also cleans up some logic related to unsupported package versions in `ignition.dsl`

## sdformat.dsl

* [Build Status](https://build.osrfoundation.org/job/_dsl_sdformat/1012/) https://build.osrfoundation.org/job/_dsl_sdformat/1012/

~~~
Added items:
    GeneratedJob{name='sdformat4-install_bottle-homebrew-amd64'}
    GeneratedJob{name='sdformat6-install_bottle-homebrew-amd64'}
    GeneratedJob{name='sdformat9-install_bottle-homebrew-amd64'}
    GeneratedJob{name='sdformat10-install_bottle-homebrew-amd64'}
    GeneratedJob{name='sdformat11-install_bottle-homebrew-amd64'}
~~~

* [Build Status](https://build.osrfoundation.org/job/sdformat11-install_bottle-homebrew-amd64/) https://build.osrfoundation.org/job/sdformat11-install_bottle-homebrew-amd64/

## ignition_collection.dsl

* [Build Status](https://build.osrfoundation.org/job/_dsl_ignition_collection/361/) https://build.osrfoundation.org/job/_dsl_ignition_collection/361/
* https://build.osrfoundation.org/view/ign-edifice/
* https://build.osrfoundation.org/view/ign-fortress/

## ignition.dsl

* [Build Status](https://build.osrfoundation.org/job/_dsl_ignition/1156/) https://build.osrfoundation.org/job/_dsl_ignition/1156/

### CODE REVIEW

File: jenkins-scripts/dsl/ignition.dsl [hunk 1]
```diff
@@ -583,12 +583,7 @@ ignition_software.each { ign_sw ->
...
     {
       // disable some bottles
-      if (("${ign_sw}" == "cmake" && "${major_version}" == "1") ||
-          ("${ign_sw}" == "common" && "${major_version}" == "2") ||
-          ("${ign_sw}" == "fuel-tools" && "${major_version}" == "2") ||
-          ("${ign_sw}" == "gui" && "${major_version}" == "0") ||
-          ("${ign_sw}" == "math" && "${major_version}" == "5") ||
-          ("${ign_sw}" == "msgs" && "${major_version}" == "2") ||
+      if (("${ign_sw}" == "gui" && "${major_version}" == "0") ||
           ("${ign_sw}" == "transport" && "${major_version}" == "5"))
```
Comment (chapulina) [inline jenkins-scripts/dsl/ignition.dsl:587]: Can't transport5 be removed too?

**Review by chapulina — APPROVED** [review]
LGTM, I just have a question about transport5.

File: jenkins-scripts/dsl/ignition.dsl (hunk 1 above)
Comment (scpeters) [reply jenkins-scripts/dsl/ignition.dsl:587]: yeah, I meant to remove that too

Comment (scpeters) [reply jenkins-scripts/dsl/ignition.dsl]: 4f384fe7d8cf7523c195b1c4ba2b8c0e0109c858

[Build Status](https://build.osrfoundation.org/job/_dsl_ignition/1157/) https://build.osrfoundation.org/job/_dsl_ignition/1157/

**Pull request merged**